packages/css-data/bin/property-value-descriptions.ts

```diff
@@ -72,12 +107,87 @@ const newPropertiesNames = Object.keys(keywordValues)
         typeof propertiesOverrides[property] !== "string")
   );
 
+const newPropertySyntaxes = propertiesAndFunctionsSyntax.filter(
+  (property) =>
+    forceRegenerate ||
+    (typeof propertySyntaxesGenerated[property] !== "string" &&
+      typeof propertySyntaxesGenerated[property] !== "string")
+);
+
 let retries = 0;
 
 const backoff = (num: number) => {
```

Comment:
inlining retry with incremental backoff is a bit too much, there are good libs for that with battle tested logic, but also maybe you don't need incremental backoff, you can just repeat fixed amount of times after fixed amount of seconds and that's it, our use case is simple here

Reply:
I used this small lib a years ago https://github.com/tim-kos/node-retry its tiny, but maybe there is something better @TrySound might know

Reply:
def. don't love the retry/backoff logic in the script here

Reply:
would love to organize this script a little better overall, its a bit hard to read, I just tried